### ISSUE TITLE
The 'indentation' rule tests are failing on Node 11

### ISSUE THREAD

**Issue by jhedstrom** [vsiakka/gherkin-lint#141]
When running tests over in #126 the latest version of node (currently 11) has broken indentation rules. I've confirmed this breaks on master too:

```
Indentation rule
    1) doesn't raise errors when the default conifguration is used and there are no indentation violations (spaces)
    2) doesn't raise errors when the default conifguration is used are and there no indentation violations (tabs)
    3) detects errors for features, backgrounds, scenarios, scenario outlines and steps (spaces)
    4) detects errors for features, backgrounds, scenarios, scenario outlines and steps (tabs)
    5) detects errors for features, backgrounds, scenarios, scenario outlines and steps in other languages
    6) defaults the tag indentation settings when they are not set
    7) observe tag indentation settings when they are overriden
```

Full log: https://travis-ci.org/vsiakka/gherkin-lint/jobs/480107338

**Comment by josemenveg** [comment]
This is a known issue, I have opened the #138 issue for the same reason.

I have [a pull](https://github.com/vsiakka/gherkin-lint/pull/137) to solve it and to add some new rules.

If it's a problem for you, it's solved in my fork.

**Comment by vsiakka** [comment]
Fixed with #161
